app/Models/Author.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Author extends Model
+{
+    use HasFactory;
+
+    protected $table = 'authors';
+    protected $guarded = [];
```

Comment:
Why leave an empty array here? Only override this property when needed. Otherwise don't use it.

Reply:
Fix in other Models too.